src/components/Scheduler/DateDetails.tsx

```diff
@@ -1,46 +1,73 @@
+import { translate } from '@/helpers/translate';
+import { MeetingItem, MeetingSchedule } from '@/types/meetings';
 import { useState } from 'react';
+import { Calendar } from '../Calendar';
 import { CaretDown } from '../Icons/CaretDown';
+import { Strike } from '../Icons/Strike';
 import { Www } from '../Icons/Www';
+import { Button } from '../common/Button';
 import { Toggler } from '../common/Toggler';
+import { AvailableTimes } from './AvailableTimes';
+
+interface Props {
+  event: MeetingItem;
+}
+
+type ToggleValues = '12h' | '24h';
+
+export const DateDetails = ({ event }: Props) => {
+  const [togglerValue, setTogglerValue] = useState<ToggleValues>('12h');
+  const [selectedDay, setSelectedDay] = useState<MeetingSchedule | null>(null);
 
-export const DateDetails = () => {
-  const [togglerValue, setTogglerValue] = useState('AM');
-  const toggleAmPm = (value: string) => {
-    setTogglerValue(value);
-  };
   return (
     <div className='border-l border-off-gray pl-7'>
       <div className='flex flex-col gap-6 py-10 pr-10'>
         <div className='flex items-start'>
           <p className='text-[22px] font-semibold leading-none tracking-[0.15px] text-gray'>
-            Select date and time
+            {translate('Select date and time')}
```

Comment:
Support for multiple language?

Reply:
> Support for multiple language?

yeah, I created this function for when we want to implement multiple languages